### ISSUE TITLE
How to define custom scalars when generating code?

### ISSUE THREAD

**Issue by njam** [graphql-rust/graphql-client#274]
I'm generating the client code with the CLI tool.

The generated file for a query "GetData" will look something like this, with a custom scalar "URL":
```rust
pub struct GetData;

pub mod get_data {

    #[doc = "URL string"]
    type URL = super::URL;
[...]
```
I'm struggling to find a way of defining those custom scalars _outside_ of the generated file. Is there a way to do that?
Or will that only be possible after https://github.com/graphql-rust/graphql-client/issues/245 ?

**Comment by tomhoule** [comment]
Good point, I haven't thought about how to do this with the CLI - I hope I'll be able to work on #245 soon. In the meantime, the derive flow works (the custom scalars need to be in the same module as the struct under derive). If it's very urgent, we could try to implement a workaround (like changing the `super` to `super::super` when generating code with the CLI. What do you think?

**Comment by njam** [comment]
Thanks for the reply!
For now I will apply a workaround locally in my project.

Please close this ticket if it's covered by #245
